crates/sncast/tests/helpers/devnet.rs

```diff
@@ -59,28 +61,10 @@ fn start_devnet() {
         }
     }
 
-    Command::new("tests/utils/build_contracts.sh")
-        .spawn()
-        .expect("Failed to compile contracts")
-        .wait()
-        .expect("Timed out compiling contracts");
-
     let rt = Runtime::new().expect("Could not instantiate Runtime");
-    rt.block_on(declare_deploy_contract(
-        "user1",
-        "/map/target/dev/map_Map",
-        "CAST_MAP",
-    ));
-    rt.block_on(declare_contract(
-        "user4",
-        "/constructor_with_params/target/dev/constructor_with_params_ConstructorWithParams",
-        "CAST_WITH_CONSTRUCTOR",
-    ));
 
     rt.block_on(deploy_keystore_account());
     rt.block_on(deploy_cairo_0_account());
```

Comment:
would it be possible to also get rid of those?

Reply:
As discussed offline, it's technically possible to eliminate this. However, we would then need to either keep the private keys of the accounts deployed to Sepolia in the repo or find a way to conceal them. Given these considerations, it seems better to leave it as it is.